radis/lbl/broadening.py

```diff
@@ -1740,7 +1740,7 @@ def _apply_lineshape(self, broadened_param, line_profile, shifted_wavenum):
         return wavenumber, sumoflines
 
     def _apply_lineshape_DLM(
-        self, broadened_param, line_profile_DLM, shifted_wavenum, wL, wG, wL_dat, wG_dat
+        self, broadened_param, line_profile_DLM, shifted_wavenum, wL, wG, wL_dat, wG_dat, awL_kind = 2, awG_kind = 2
```

Comment:
I would remove the default values here for clarity (they're overwritten by the default value in SpectrumFactory() anyway )